installer/scripts/service_control

```diff
@@ -45,6 +46,14 @@ is_omsagent_running()
     fi
 }
 
+exit_if_agent_not_onboarded()
+{
+    if [ ! -f $OMSADMIN_CONF ]; then
+        echo "Warning: Agent is not onboarded. omsagent cannot be registered as a service."
+        exit 1
```

Comment:
I'm not sure this is a good idea. Did you build a shell bundle, and does this work properly?

The concern is that this can trigger an installation failure (since we're exiting the script with an error) even though this is not actually an error, it's a perfectly normal condition (and frankly, and expected one).

Did you consider emitting the warning and exiting zero? Or, better yet, just return (like 'return') from the routine and trap it from the caller? (Not sure if that's useful, though).